pkg/sharings/recipients.go

```diff
@@ -0,0 +1,42 @@
+package sharings
+
+import (
+	"github.com/cozy/cozy-stack/pkg/consts"
+	"github.com/cozy/cozy-stack/pkg/oauth"
+	"github.com/cozy/cozy-stack/web/jsonapi"
+)
+
+// Recipient is a struct describing a sharing recipient
+type Recipient struct {
+	RID    string `json:"_id,omitempty"`
+	RRev   string `json:"_rev,omitempty"`
+	Email  string `json:"email"`
+	URL    string `json:"url"`
+	Client *oauth.Client
+}
+
+// ID returns the recipient qualified identifier
+func (r *Recipient) ID() string { return r.RID }
+
+// Rev returns the recipient revision
+func (r *Recipient) Rev() string { return r.RRev }
+
+// DocType returns the recipient document type
+func (r *Recipient) DocType() string { return consts.Recipients }
+
+// SetID changes the recipient qualified identifier
+func (r *Recipient) SetID(id string) { r.RID = id }
+
+// SetRev changes the recipient revision
+func (r *Recipient) SetRev(rev string) { r.RRev = rev }
+
+// Relationships implements jsonapi.Doc
+func (r *Recipient) Relationships() jsonapi.RelationshipMap { return nil }
```

Comment:
Yes, but it's not obvious when or if it will be needed. Should I remove it for now?